source/beamline/VESPERS/VESPERSEndstation.h

```diff
@@ -111,7 +114,7 @@ public slots:
 	/// Sets the current control that should be focused on.  It will find the correct control based on the name.  Control is set to 0 if invalid name is given.
 	void setCurrent(QString name);
 	/// Toggles the current state of the laser power.
-	void toggleLaserPower() { toggleControl(laserPower_); emit laserPoweredChanged(); }
+	void toggleLaserPower() { laserPower_->move(1); emit laserPoweredChanged(); }
```

Comment:
Couple of questions:
- Are there other beamlines that use these kind of toggle pvs, and if so is there a case to be made for VESPERSToggleControl to be CLSToggleControl?
- If the laser power is toggled in the fashion, can it be a VESPERSToggleControl (or is there some detail of the toggle control/laser behaviour that I'm missing)?